tests/plugin-validation.spec.js

```diff
@@ -50,6 +50,30 @@ const tests = [
       expectedErrMessage: 'files[1] should be an non-empty string',
     },
   ],
+  [
+    'pattern is empty',
+    {
+      options: {
+        group: [{
+          pattern: [],
+          to: 'outpath',
+        }],
+      },
+      expectedErrMessage: 'pattern should be an non-empty array',
+    },
+  ],
+  [
+    'pattern has an empty string',
+    {
+      options: {
+        group: [{
+          pattern: ['pattern', ''],
+          to: 'outpath',
+        }],
+      },
+      expectedErrMessage: 'pattern[1] should be an non-empty string',
```

Comment:
```suggestion
      expectedErrMessage: 'pattern\\[1\\] should be an non-empty string',
```

Please pull the recent changes from upstream